bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -20,6 +20,7 @@ from airflow.providers.google.cloud.sensors.bigquery import (
 from airflow.sensors.external_task import ExternalTaskMarker
 from airflow.sensors.external_task import ExternalTaskSensor
 from airflow.utils.task_group import TaskGroup
+from airflow.providers.cncf.kubernetes.secret import Secret
```

Comment:
Can this import be optional?
```suggestion
{% if task.secrets|length > 0 %}
from airflow.providers.cncf.kubernetes.secret import Secret
{% endif %}
```

Reply:
added some logic using the existing namespace since this needs to handle a list of tasks